api/src/formatters/csv.ts

```diff
@@ -29,9 +29,32 @@ export const csvParser: Parser = async (data: string) => {
   };
 };
 
+const csvInjectionProtector = (str: string) => {
+  const riskyChars = ["=", "+", "-", "@", ",", ";", "0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
+
+  if(!str) return "";
+
+  const firstChar = str.charAt(0);
+  const isInjected = riskyChars.includes(firstChar);
+  if(!isInjected) return str;
+
+  const sliceStr = str.slice(1);
+  return csvInjectionProtector(sliceStr);
+}
+
 export const csvExporter: Exporter = async (data: IntermediateTranslationFormat) => {
+  
+  // clear some characters
+  const protectedTerm = csvInjectionProtector(data.translations[0].term);
+  const protectedTranslation = csvInjectionProtector(data.translations[0].translation);
+
+  const payload = [{
+    term: protectedTerm.replace('0x0d', ''), 
```

Comment:
@bvenceslas what is "0x0d"?  Why you did not add it into `csvInjectionProtector` function?